crates/rolldown/tests/esbuild/loader/loader_json_common_js_and_es6/artifacts.snap

```diff
@@ -0,0 +1,45 @@
+---
+source: crates/rolldown_testing/src/case/case.rs
+expression: content
+input_file: crates/rolldown/tests/esbuild/loader/loader_json_common_js_and_es6
+---
+# Assets
+
+## entry_js.mjs
+
+```js
+
+
+//#region y.json
+const y1 = true;
+const y2 = false;
+var y_default = {
+	y1,
+	y2
+};
+
+//#endregion
+//#region z.json
+const small = "some small text";
+const key_2 = "test keyword imports";
+
+//#endregion
+//#region x.json
+var x_ns, x, x_default;
+var init_x = __esmMin(() => {
+	x_ns = {};
+	__export(x_ns, {
+		default: () => x_default,
+		x: () => x
+	});
+	x = true;
+	x_default = {x};
+});
+
+//#endregion
+//#region entry.js
+const x_json = (init_x(), (__toCommonJS(x_ns)).default);
+console.log(x_json, y_default, small, key_2);
+
+//#endregion
```

Comment:
```js
================================================================================
TestLoaderJSONCommonJSAndES6
---------- /out.js ----------
// x.json
var require_x = __commonJS({
  "x.json"(exports, module) {
    module.exports = { x: true };
  }
});

// y.json
var y_default = { y1: true, y2: false };

// z.json
var small = "some small text";
var if2 = "test keyword imports";

// entry.js
var x_json = require_x();
console.log(x_json, y_default, small, if2);
```

It might also be a problem mentioned in #1740.

Reply:
revert this please, this are too much difference.